.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        scala: [2.12.13, 2.13.5, 3.0.0]
+        scala: [2.12.13, 2.13.5, 3.1.0]
```

Comment:
See https://github.com/scala-steward-org/scala-steward/issues/2467 - shouldn't we stay on 3.0.x, or add that flag?